apps/parsley/src/constants/enums.ts

```diff
@@ -2,7 +2,8 @@ enum LogTypes {
   EVERGREEN_TASK_FILE = "EVERGREEN_TASK_FILE",
   EVERGREEN_TASK_LOGS = "EVERGREEN_TASK_LOGS",
   EVERGREEN_TEST_LOGS = "EVERGREEN_TEST_LOGS",
-  RESMOKE_LOGS = "RESMOKE_LOGS",
+  EVERGREEN_ALL_LOGS = "EVERGREEN_ALL_LOGS",
```

Comment:
this also makes me think of All Logs for Evergreen task logs, maybe it can have a more specific name